### DISCUSSION TITLE
LAB-B  06-cholesky-accelerate module1  project set up

### DISCUSSION THREAD

**Discussion by chaoNTUEE** [bol-edu/HLS-SOC-Discussions#33] [Q&A] [answered]
I run the "make run TARGET=sw_emu" command from vitis tutorial to set up my project, but there is an error:
![image](https://user-images.githubusercontent.com/115763094/196924254-2b609fc8-a24c-4c49-84d4-8098436164ec.png)

Then I try to use GUI to build up project, but encounter another error:
![image](https://user-images.githubusercontent.com/115763094/196925132-a34d3dcd-1454-44fb-986c-777beb0df62f.png)

Does anyone else meet this problem?

**Comment by BrianEE07** [comment, marked as answer]
I have the same problem, too. A solution I found is to build an application project from scratch using GUI just like Lab3.
Some tips:
1. The files to be imported for kernel or host can be found in provided Makefile
2. Run config program argument I used: -xclbin ./binary_container_1.xclbin -runs 1 -M 512 -N 512 -seed 12

**Reply by chaoNTUEE** [reply]
Thanks a lot!  Tip 1 works for me.
